core/src/host_imports/lowering/lo_helper.rs

```diff
@@ -55,19 +55,23 @@ impl<
 }
 
 impl<
-        's,
         WB: WasmBackend,
         MV: MemoryView<DelayedContextLifetime<WB>>,
         M: Memory<MV, DelayedContextLifetime<WB>>,
-    > Allocatable<MV, DelayedContextLifetime<WB>> for LoHelper<'s, WB, MV, M>
+    > Allocatable<MV, DelayedContextLifetime<WB>> for LoHelper<WB, MV, M>
 {
-    fn allocate(
-        &mut self,
-        store: &mut <WB as WasmBackend>::ContextMut<'_>,
+    fn allocate<'this, 'ctx1: 'this, 'ctx2: 'this>(
```

Comment:
let's use more meaningful names than `ctx1` and `ctx2`, smth like `store`, `ctx`.